### ISSUE TITLE
Md5 drive error

### ISSUE THREAD

**Issue by Gilgamesh0028** [Bionus/imgbrd-grabber#3167]
**Bug description**

**Steps to reproduce**

android 13, latest release

try and view MP4:

error in log: 
![Screenshot_20240519-141331](https://github.com/Bionus/imgbrd-grabber/assets/98428877/f2a98b07-e107-4e63-a3d3-0c3a0293fde2)

**Expected behavior**

unable to load video

**Context**

* fresh install, nothing was changed from default asides from light mode->dark mode, and pasting a few tags into the blacklist, then selecting to hide images with blacklisted tags.
can't figure out how r add logs from mobile GitHub client. please advise on solution.

**Comment by Bionus** [comment]
Indeed. I guess the SQLite driver is missing from the new APK, I'll check it out.
